AnagramSolver.Cli/Program.cs

```diff
@@ -9,45 +9,46 @@ class Program
     {
         static void Main(string[] args)
         {
-            var builder = new ConfigurationBuilder()
-               .AddJsonFile($"appsettings.json", true, true);
-            var config = builder.Build();
-            var minLength = config["MinWordLength"];
-            int minLengthAsNumber = Int32.Parse(minLength);
-            var maxAnagrams = config["MaxNumberOfAnagrams"];
-            int maxAnagramsAsNumber = Int32.Parse(maxAnagrams);
-            Console.WriteLine(maxAnagramsAsNumber);
-
+            int minLength, maxAnagrams;
+            string filePath;
+            ConfigureAppSettings(out minLength, out maxAnagrams, out filePath);
             var result = new BusinessLogic.Classes.AnagramSolver();
-            var command = "";
-            
+            var command = string.Empty;
 
-            while(true)
+
+            while (true)
             {
                 Console.WriteLine("Iveskite zodi kurio anogramos norite arba iveskite exit jei norite iseiti is konsoles");
                 command = Console.ReadLine();
-                if(command.Length < minLengthAsNumber)
+                if (command.Length < minLength)
                 {
                     Console.WriteLine($"Zodis turi buti bent is {minLength} raidziu");
                     command = Console.ReadLine();
                 }
-                if(command == "exit")
+                if (command == "exit")
                 {
                     break;
                 }
-                var listOfAnagrams = result.GetAnagrams(command);
+                var listOfAnagrams = result.GetAnagrams(command, minLength, filePath);
```

Comment:
This is fine for now, however if we apply more configuration in appsettings.json we have to append new parameter for each line. A method with more than three parameters are questionable.

`Bob Martin says the ideal number of parameters is 0 and that 3 is stretching it.`
(https://www.informit.com/articles/article.aspx?p=1375308)

Strongly recommend to check for better practices (later).
(https://stackoverflow.com/questions/48948905/how-to-access-appsettings-from-another-project/48951371)

Reply:
okay, will do